modules/ROOT/pages/legacy-search-experience.adoc

```diff
@@ -0,0 +1,311 @@
+
+Example search terms:
+
+[source,text,linenums]
+----
+sales:connect
+Sales-Connect
+connect:/sales
+-connect -sales
+----
+
+This table shows how each search term is interpreted based on the values in the asset name,
+asset ID, and tags for an asset. If the search term maps to the values correctly, a match occurs. If not,
+the reason is provided.
+
+[%header%autowidth.spread]
+|===
+|Asset Name |Asset ID |Tags |Matches?
+|Salesforce API |salesforce-api |mule-Connector |Yes
+|Salesforce Connector |salesforce-connector |REST |Yes
+|Validations API |validations-api |salesforce-connecting |Yes
+|Optimized API |optimized-api |resource:/connect, salesforce:enabled |Yes
+|Cisco Sales API  |cisco-sales-api |Connecting, Productive |Yes
+|Cisco Connector |cisco-connector |com.cisco.connector.sales
+|Yes, because the tag is split into separate terms.
+|Salesforce API |salesforce-api |muleconnector, MuleConnector |No. `connect` was not found.
+|Salesforce Connec |salesforce-connec |REST |No. `connect` was not found.
+|Validations API |validations-api |connector |No. `sales` was not found.
+|Cisco Presales API |cisco-presales-api |Connector, api |No. `sales` was not found.
+|===
+
+=== Phrase Search Separators
+
+Use phrase search to find a group of terms inside quotation marks. Phrase searches match an asset only if all the exact terms inside quotes appear in consecutive order.
+
+If you input any of these searches in Exchange, Exchange replaces non-alphanumeric symbols with spaces before searching for assets.
+
+Example search terms:
+
+[source,text,linenums]
+----
+Api: "Sales connect"
+Api "Sales-connect"
+Api "Sales:/connect"
+Api-"Sales/connect"
+"Sales connect":Api
+----
+
+This table shows how each search term is interpreted based on the values in the asset name,
+asset ID, and tags for an asset. If the search term maps to the values correctly, a match occurs. If not,
+the reason is provided.
+
+[%header%autowidth.spread]
+|===
+|Asset Name |Asset ID |Tags |Matches?
+|Salesforce API |salesforce-api |mule-Connector |No. `sales connect` was not found.
+|Salesforce Connector |salesforce-connector |REST |No
+|Sales Connect Asset |validations-api |mule-connector |Yes
+|Sales Connect Images |validations-images |mule-connector |No. `api` was not found.
+|Optimized API |optimized-api |resource:/connect, sales:enabled |No. `sales connect` was not found.
+|Optimized API |optimized-api |resource:/sales, connect:enabled |No. `sales connect` was not found, because each term is in different tags.
+|Optimized API |optimized-api |sales:/connect |Yes
+|===
+
+== Search by Types, Tags, Categories, and Custom Fields 
+
+You can enter search text to search for assets. Click *All Types* to set a filter for specific asset types, or combine search text with the filters listed in the type menu.
```

Comment:
Should type be init cap and bold?

Reply:
Yes, thanks for pointing this out. Should be *All types* menu.